Commands/Publishing/AddWikiPage.cs

```diff
@@ -10,7 +10,7 @@ namespace SharePointPnP.PowerShell.Commands.Publishing
         Category = CmdletHelpCategory.Publishing)]
     public class AddWikiPage : SPOWebCmdlet
     {
-        [Parameter(Mandatory = true)]
+        [Parameter(Mandatory = true, HelpMessage = "The server relative page URL")]
```

Comment:
maybe good to show an example of a server relative page url, like "/sites/demo1/sitepages/mypage.aspx"